tests/scripts/all.sh

```diff
@@ -1361,6 +1361,9 @@ component_test_zeroize () {
     unset gdb_disable_aslr
 }
 
+support_check_python_files () {
+    type pylint3 >/dev/null 2>/dev/null
```

Comment:
We need to make sure we have pylint3 in our CI.

Reply:
At the moment we have neither `pylint` nor `pylint3`. If we had `pylint`, the runs would fail.

Reply:
Since you are fixing things in this PR, it'd be a shame to regress.